request/request.go

```diff
@@ -137,3 +115,15 @@ func Request(namespace, manifestDir, sha, githubURL, apiURL, org, repo, token st
 		log.WithField("pullRequest", *pr.Number).Info("pull request raised")
 	}
 }
+
+func git(workingDir string, args ...string) (err error) {
+	log.Info("git", args)
+	cmd := exec.Command("git", args...)
+	cmd.Env = os.Environ()
+	cmd.Dir = workingDir
+	err = cmd.Run()
+	if err != nil {
+		log.WithError(err).Error("executing git with args: %s", args)
```

Comment:
This error logging is probably not necessary because we log the `args` in the beginning of the function and log the error after we called the function.